src/lib.rs

```diff
@@ -72,6 +73,7 @@ pub fn build(
     plan_path: Option<String>,
     out_dir: Option<String>,
     tags: Vec<&str>,
+    labels: Vec<&str>,
```

Comment:
Why are we using `&str` as opposed to `String`?

Reply:
No big reason. It is just the type that comes from clap's `values_of` https://github.com/railwayapp/nixpacks/blob/main/src/main.rs#L119